sql/createTables.sql

```diff
@@ -1,8 +1,84 @@
-DROP TABLE IF EXISTS food;
+DROP TABLE  IF EXISTS comments;
```

Comment:
Slightly confused about the order of the queries.

Understand that if you wanted to restart the project from scratch you would have to drop the tables dependent on foreign keys first but it seems odd that the query then immediately repopulates that table?

Would have to create the User table first then the resources then the comments, etc due to foreign key restraints. 

Suggestion would be to have some clear order either through different files or understandable formatting.